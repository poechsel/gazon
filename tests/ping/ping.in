login u1
pass p1
date
ping 8.8.8.8
ping epfl.ch
ping nonexistingaddress
date 
