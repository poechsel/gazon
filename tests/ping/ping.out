<line>Tue Nov 5 16:00:00 CEST 2019
PING 8.8.8.8 (8.8.8.8) 56(84) bytes of data.
64 bytes from 8.8.8.8: icmp_seq=1 ttl=<*> time=<*>ms

--- 8.8.8.8 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time <*>ms
rtt min/avg/max/mdev = <*>ms

PING epfl.ch (<*>) 56(84) bytes of data.
64 bytes from www-origin.epfl.ch (<*>): icmp_seq=1 ttl=<*> time=<*>ms

--- epfl.ch ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time <*>ms
rtt min/avg/max/mdev = <*>ms
ping: nonexistingaddress: Name or service not known
<line>Tue Nov 5 16:00:15 CEST 2019 
